.ci/Jenkinsfile_coverage

```diff
@@ -5,89 +5,26 @@ kibanaLibrary.load() // load from the Jenkins instance
 
 kibanaPipeline(timeoutMinutes: 240) {
   catchErrors {
+    def timestamp = new Date(currentBuild.startTimeInMillis).format("yyyy-MM-dd'T'HH:mm:ss'Z'", TimeZone.getTimeZone("UTC"))
```

Comment:
context here?  mostly where to verify the change.  overall timestamps are good

Reply:
It's a canonical time stamp for the entire build...lots of pieces use this value.  It's placed in the metadata that is ingested, it's added to the index for (latest hyperlinks), it's in the coverage html files themselves, etc.  It's basically how each data point is collated.

Reply:
<img width="1680" alt="Screen Shot 2020-05-06 at 11 38 28 AM" src="https://user-images.githubusercontent.com/4858051/81210011-7d632900-8f8e-11ea-8350-721cc021fdec.png">

Reply:
<img width="1307" alt="Screen Shot 2020-05-06 at 11 41 42 AM" src="https://user-images.githubusercontent.com/4858051/81210117-a5528c80-8f8e-11ea-9cac-9734775dcf11.png">

Reply:
<img width="1342" alt="Screen Shot 2020-05-06 at 11 44 04 AM" src="https://user-images.githubusercontent.com/4858051/81210336-fc586180-8f8e-11ea-8c57-53c685cf59d2.png">
It's in all-the-things :)

Reply:
<img width="1678" alt="Screen Shot 2020-05-06 at 11 48 01 AM" src="https://user-images.githubusercontent.com/4858051/81210694-7983d680-8f8f-11ea-94c7-ee68bd7bec93.png">
Also here

Reply:
We are just using a here doc to populate those latest links.  This will prolly get changed in the future tbh

Reply:
got it! thanks